clients/admin-ui/src/features/datamap/datamap.slice.ts

```diff
@@ -104,6 +107,60 @@ const datamapApi = baseApi.injectEndpoints({
       },
       providesTags: ["Datamap"],
     }),
+    exportMinimalDatamapReport: build.mutation<
+      Page_DatamapReport_,
+      {
+        groupBy: DATAMAP_GROUPING;
+        pageIndex: number;
+        pageSize: number;
+        search: string;
+        dataUses?: string;
+        dataCategories?: string;
+        dataSubjects?: string;
+        format?: ExportFormat;
+      }
+    >({
+      query: ({
+        groupBy,
+        pageIndex,
+        pageSize,
+        search,
+        dataUses,
+        dataCategories,
+        dataSubjects,
+        format,
+      }) => {
+        let queryString = `page=${pageIndex}&size=${pageSize}&group_by=${groupBy}`;
+        if (dataUses) {
+          queryString += `&${dataUses}`;
+        }
+        if (dataCategories) {
+          queryString += `&${dataCategories}`;
+        }
+        if (dataSubjects) {
+          queryString += `&${dataSubjects}`;
+        }
+        if (search) {
+          queryString += `&search=${search}`;
+        }
+        return {
+          url: `plus/datamap/minimal/${format}?${queryString}`,
```

Comment:
We specifically have `admin-ui/src/features/plus/plus.slice.ts` where we call plus endpoints, and I know we already have a call to /plus in this file before your changes, so I'm wondering if it makes sense to refactor in the future to keep plus calls centralized in `plus.slice.ts`?